### PR TITLE
[DOCFIX] Validate `Tensorflow.md`

### PR DESCRIPTION
**Pull request by jenoudet**
Validate `Tensorflow.md` doc.

### CODE REVIEW

File: docs/en/compute/Tensorflow.md [hunk 1]
```diff
@@ -35,22 +35,22 @@ and allow Tensorflow applications to access the data through Alluxio POSIX API.
 
 Run the following command to install FUSE on Linux:
 
-```
+```shell
```
Comment (LuQQiu) [inline docs/en/compute/Tensorflow.md:38]: we have the annotation `console` other than `shell` in other docs, not quite sure which is the convention

**Review by LuQQiu — APPROVED** [review]
LGTM, left one small comment about whether to use `console` or `shell` in our docs

File: docs/en/compute/Tensorflow.md (hunk 1 above)
Comment (jenoudet) [reply docs/en/compute/Tensorflow.md:38]: In my experience, I find that `console` colors text blocks inconsistently. It sometimes leaves line italicized and greyed out when it is not necessary. I find that `shell` does a better job with readability and coloring consistency, but I can change it back if `console` is the convention.

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.8 please

**Pull request merged**